test/local/lib/senders/translator.js

```diff
@@ -2,25 +2,37 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+'use strict'
+
 const assert = require('insist')
 
 require('../../../../lib/senders/translator')(['en', 'pt_br', 'DE', 'ES_AR', 'ES_cl'], 'en')
-.then(
-  function (translator) {
-    it(
-      'translator works with upper and lowercase languages',
-      function (t) {
-        var x = translator('PT-br,DE')
-        assert.equal(x.language, 'pt-BR')
-        x = translator('bu-ll,es-ar')
-        assert.equal(x.language, 'es-AR')
-        x = translator('es-CL')
-        assert.equal(x.language, 'es-CL')
-        x = translator('en-US')
-        assert.equal(x.language, 'en')
-        x = translator('db-LB') // a locale that does not exist
-        assert.equal(x.language, 'en')
-      }
-    )
-  }
-)
+.then(translator => {
+  it('returns the correct interface', () => {
+    assert.equal(typeof translator, 'object')
+    assert.equal(Object.keys(translator).length, 2)
+    assert.equal(typeof translator.getTranslator, 'function')
+    assert.equal(typeof translator.getLocale, 'function')
+  })
+
+  it('getTranslator works with upper and lowercase languages', () => {
+    let x = translator.getTranslator('PT-br,DE')
+    assert.equal(x.language, 'pt-BR')
+    x = translator.getTranslator('bu-ll,es-ar')
+    assert.equal(x.language, 'es-AR')
+    x = translator.getTranslator('es-CL')
+    assert.equal(x.language, 'es-CL')
+    x = translator.getTranslator('en-US')
+    assert.equal(x.language, 'en')
+    x = translator.getTranslator('db-LB') // a locale that does not exist
```

Comment:
Fwiw, I made a change so that these locales will show up as `en.default` rather than `en`. Thanks for pointing it out!